efficiency/misc/make_html.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
Do we need this file? I just see personal directories, so I am not sure we want it if it's only possible to run it on specific machines.

Reply:
this is the issue with the pandoc that is not installed on lnx7188.
Perhaps what we can do is to have a separate git repo for "LSTPerformanceWebpage" (or some better name)
and have them as a submodule.
And I can update the repo separate from our code.

Reply:
I think I got the whole concept of the html creation. I don't have any specific preference, so I leave it up to you to handle it as you prefer. Resolving the comment, thanks for the explanations.